### ISSUE TITLE
Uncaught TypeError: atom.workspace.getLeftDock(...).observeVisible is not a function

### ISSUE THREAD

**Issue by revelt** [lexicalunit/no-title-bar#29]
[Enter steps to reproduce:]

After updating to the latest version this error started popping up. Something must have changed in the latest Atom API's to cause this:

![error_popup](https://user-images.githubusercontent.com/8344688/29310377-6e1337dc-81a5-11e7-9f88-d4702bb9374f.png)

---

**Atom**: 1.19.1 x64
**Electron**: 1.6.9
**OS**: Mac OS X 10.12.6
**Thrown From**: [no-title-bar](https://github.com/lexicalunit/no-title-bar) package 1.7.1

### Stack Trace

Uncaught TypeError: atom.workspace.getLeftDock(...).observeVisible is not a function

```
At /Users/royston/.atom/packages/no-title-bar/lib/no-title-bar.js:64

TypeError: atom.workspace.getLeftDock(...).observeVisible is not a function
    at finishInitialize (/packages/no-title-bar/lib/no-title-bar.js:64:34)
    at /packages/no-title-bar/lib/no-title-bar.js:37:5)
```

### Commands

```
     -0:34.2.0 tree-view:show (atom-workspace.workspace.scrollbars-visible-always)
```

### Non-Core Packages

```
ariake-dark-syntax 0.1.3 
atom-clock 0.1.13 
atom-easy-jsdoc 4.12.1 
atom-panic-palette-syntax 1.0.0 
atom-pug 0.2.0 
atom-react-snippets 0.2.2 
atom-wrap-in-tag 0.6.0 
aurora-theme 1.2.2 
auto-id-class 0.3.1 
auto-update-packages 1.0.1 
behave-theme 1.7.0 
browse 1.6.1 
busy-signal 1.4.3 
chester-atom-syntax 0.3.0 
close-tags 0.6.0 
color-town-syntax 0.1.4 
declarations 1.1.3 
declarations-js 1.0.2 
editorconfig 2.2.2 
file-icons 2.1.10 
flatwhite-syntax 0.8.1 
flow-ide 1.8.1 
gloom 0.6.4 
guess-indent 0.1.0 
hey-pane 1.0.0 
highlight-bad-chars 1.0.7 
hyperclick 0.1.3 
intentions 1.1.5 
jumpy 3.1.3 
language-babel 2.71.0 
language-diff 0.7.0 
language-markdown 0.25.1 
language-nunjucks 4.1.0 
language-pug 0.0.21 
linter 2.2.0 
linter-eslint 8.2.1 
linter-htmlhint 1.3.3 
linter-javac 1.9.4 
linter-js-standard 3.9.3 
linter-js-standard-engine 2.1.0 
linter-jsonlint 1.3.0 
linter-scss-lint 3.1.0 
linter-tidy 2.3.1 
linter-ui-default 1.6.5 
linter-write-good 0.9.0 
linter-xo 0.23.1 
markdown-preview-enhanced 0.14.7 
mixed-indent-warning 0.3.0 
modern-javascript-snippets 1.1.0 
monokai 0.24.0 
multi-cursor-increment 0.1.1 
no-title-bar 1.7.1 
nord-atom-syntax 0.9.1 
nord-atom-ui 0.11.0 
nucleus-dark-ui 0.12.3 
oceanic-next 1.0.0 
project-manager 3.3.5 
react 0.16.2 
sepia-syntax 1.1.0 
set-syntax 0.3.2 
simple-drag-drop-text 0.3.4 
split-diff 1.4.3 
standard-formatter 2.8.0 
tab-control 0.6.10 
text-manipulation 0.6.0 
webbox-color 0.5.9 
```

**Comment by lexicalunit** [comment]
The latest version of `no-title-bar` is only available for Atom versions `>=1.20.0 <2.0.0`. As defined [in the `package.json` file on line 15](https://github.com/lexicalunit/no-title-bar/blob/master/package.json#L15). The `observeVisible` function is only [available in Atom 1.20 for now](https://github.com/atom/atom/pull/14736).

`apm` shouldn't have let you install the latest version of `no-title-bar` as `1.20.0` is still in beta.

Do you have both Atom and Atom Beta installed? If so, installing `no-title-bar` using `apm-beta` could cause this issue if you then open Atom 1.19.x since both Atom and Atom Beta will share a directory for installed packages by default.

If you want to have separate package installations for Atom and Atom Beta you may be able to [leverage the `ATOM_HOME` environment variable](https://github.com/atom/atom/pull/5385).

**Comment by lexicalunit** [comment]
@revelt you can downgrade to a version suitable for Atom 1.19.x using:

```shell
$ apm uninstall no-title-bar
$ apm install no-title-bar@1.5.0
```
